### DISCUSSION TITLE
blog/posts/230226-gatsby-10-codeblock/

### DISCUSSION THREAD

**Discussion by giscus** [ricale/blog#86] [Giscus]
# blog/posts/230226-gatsby-10-codeblock/

이전 글들에서는 Gatsby 기본 기능을 구현을 위한 튜토리얼을 소개하며 태그 기능을 구현했고, MDX 글 안에 인라인 이미지와 테이블을 삽입했으며, 시리즈 기능도 만들어보았고, 임시글 기능도 구현했고, Google Analytics 도 적용해보았으며…

https://ricale.kr/blog/posts/230226-gatsby-10-codeblock/

**Comment by yanggwi** [comment]
훌륭하네요!

**Reply by ricale** [reply]
감사합니다 :smile:

**Comment by geunskoo** [comment]
좋은 정보 감사합니다 ㅎㅎㅎ
